llvm/tools/llvm-symbolizer/Opts.td

```diff
@@ -55,6 +55,7 @@ def pretty_print : F<"pretty-print", "Make the output more human friendly">;
 defm print_source_context_lines : Eq<"print-source-context-lines", "Print N lines of source file context">;
 def relative_address : F<"relative-address", "Interpret addresses as addresses relative to the image base">;
 def relativenames : F<"relativenames", "Strip the compilation directory from paths">;
+def skip_line_zero : F<"skip-line-zero","Show approximate non-zero line number information nearest to before the given address.">;
```

Comment:
As noted by @dwblaikie the help text should reflect what's in the docs for this option. I suggest:
> If an address does not have an associated line number, use the last line
> number from the current sequence in the line-table.

Reply:
Yes, done.